foundry/loaders/tf_wrapper.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+from tensorflow.keras.utils import Sequence
+
+class TensorflowSequence(Sequence):
+    """Foundry Dataset Converted to Tensorflow Format"""
+
+    def __init__(self, inputs, targets):
+        self.inputs=inputs
+        self.targets=targets
+
+    def __len__(self):
+        return len(self.inputs[0])
+
+    def __getitem__(self, idx):
+        item = {"input": [], "target": []}
+        
+        for input in self.inputs:
```

Comment:
Please document what these for loops are doing. I'm still dubious about returning a single array that contains multiple inputs is correct here, so let's at least explain.

The ideal solution would be to test with datasets that represent what we have in foundry. @ascourtas , does anything come to mind?

Reply:
yes the iris data used in the test files (should be in the right format), or the Atom Position Finding dataset (HDF5) plus the Zeolite dataset (tabular) would give one a comprehensive test